pom.xml

```diff
@@ -37,14 +37,6 @@
     </pluginRepository>
   </pluginRepositories>
   <build>
-      <resources>
-          <resource>
-              <directory>servers/</directory>
-              <excludes>
-                  <exclude>*</exclude>
-              </excludes>
-          </resource>
-      </resources>
```

Comment:
Revert [Exclude servers from project in pom.xml to avoid indexing](https://github.com/JetBrains/TeamCity.SonarQubePlugin/commit/c920034f3d8365296739e7a03332f7719f40ea31#diff-600376dffeb79835ede4a0b285078036), see [here](https://github.com/JetBrains/TeamCity.SonarQubePlugin/issues/76#issue-567008314)